frontend/src/components/inputs/EntityForm.vue

```diff
@@ -6,6 +6,20 @@
       <TextInput label="Slug" v-model="editEntity.slug" :args="{ id: 'slug', maxlength: 128 }"
                  :required="true"/>
       <h3 class="mt-3">Attributes</h3>
+      <div v-if="attributes" class="border-bottom border-top border-light my-2">
+        <div class="fw-bold my-2">Values to copy:</div>
+        <div v-for="attr in schema?.attributes || []" :key="`${attr.name}-${attr.id}`" class="mb-1">
+          <label :class="`${attributes[attr.name] ? 'cursor-pointer' : ''}`">
+            {{ attr.name }}
+            <sup v-if="requiredAttrs.includes(attr.name)" class="text-danger me-1"
+                 data-bs-toggle="tooltip" title="This value is required">*</sup>
+            <input type="checkbox" @change="onAttributeCopyChange($event.target.checked, attr.name)"
+                   :checked="editEntity[attr.name]"
+                   :disabled="!attributes[attr.name]"
+                   :class="`${attributes[attr.name] ? 'cursor-pointer' : ''}`" />
```

Comment:
superfluous string format? `:class="attributes[attr.name] ? 'cursor-pointer' : ''"` should do the trick, right?